tasks/workflow/builder/builder.go

```diff
@@ -38,6 +38,10 @@ func BuildWorkFlow(kv *api.KV, deploymentID, wfName string) (map[string]*Step, e
 		return nil, err
 	}
 
+	if wf.Steps == nil || len(wf.Steps) == 0 {
+		return nil, tasks.NewInconsistentDeploymentError(deploymentID)
```

Comment:
Why does this lives in `tasks` packages and not in `deployments` package?

Reply:
Honestly, I didn't know where to put this !

Reply:
Done